api/client.cpp

```diff
@@ -93,7 +92,7 @@ error::Error Client::AsyncCall(
 						header_handler(ex_resp);
 						return;
 					}
-					logger_.Debug(
```

Comment:
Q: Why?

Reply:
Because there's no `logger_` any more. It could do `http_client_.logger_.Debug(...)`, but that would require `logger_` to be `public` and it's not worth the ugly change, I think.